lib/screens/home/tabs/contact_us/contact_us.dart

```diff
@@ -27,23 +33,312 @@ class ContactUsScreen extends StatelessWidget {
+                                errorStyle: TextStyle(fontSize: 0.1),
+                                labelText: "Feedback",
+                              ),
+                            ),
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                  Container(
+                    padding: EdgeInsets.only(right: D.horizontalPadding, top: 40),
+                    alignment: Alignment.topRight,
+                    child: Container(
+                      decoration: BoxDecoration(
+                        borderRadius: BorderRadius.all(Radius.circular(30)),
+                        boxShadow: [
+                          BoxShadow(
+                            color: C.authButtonColor.withOpacity(0.2),
+                            blurRadius: 10,
+                            spreadRadius: 3,
+                            offset: Offset(0, 12),
+                          ),
+                        ],
+                      ),
+                      child: RaisedButton(
+                        shape: RoundedRectangleBorder(
+                          borderRadius: BorderRadius.all(
+                            Radius.circular(30),
+                          ),
+                        ),
+                        color: C.authButtonColor,
+                        onPressed: () => _postFeedback(context),
+                        child: Container(
+                          height: 50,
+                          width: 120,
+                          alignment: Alignment.center,
+                          child: Row(
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            children: [
+                              Text(
+                                "Submit",
+                                style: TextStyle(
+                                  color: C.primaryUnHighlightedColor,
+                                  fontSize: 20 * heightFactor,
+                                  fontWeight: FontWeight.w300,
+                                  shadows: [
+                                    Shadow(
+                                      color: Colors.black,
+                                      offset: Offset(0, 0.5),
+                                      blurRadius: 3,
+                                    ),
+                                  ],
+                                ),
+                              ),
+                            ],
+                          ),
+                        ),
+                      ),
+                    ),
+                  ),
+                ],
+              ),
+            ),
+          )
+        ],
+      ),
+    );
+  }
+
+  _launchURL(String url) async {
+    if (await canLaunch(url)) {
+      await launch(url);
+    } else {
+      throw 'Could not launch $url';
+    }
   }
 
+  String _validator(String email) => email.isEmpty ? "" : null;
```

Comment:
1. `email`?
2. No need to create a function. place the body directly in the widget as argument.